src/components/InputField/index.js

```diff
@@ -12,9 +12,18 @@ const INPUT_PROP_TYPE = PropTypes.shape({
 })
 
 const META_PROP_TYPE = PropTypes.shape({
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string),
+  ]),
 })
 
+const parseError = (error) => {
+  if (!Array.isArray(error)) {
+    return error
+  }
+  return error.length > 0 ? error[0] : undefined
+}
```

Comment:
i think i'd rather prefer the array logic to be in the conditional, and then anything else is "pass through as usual"

```
const parseError = (error) => {
  if (Array.isArray(error)) {
    return error[0]
  }
  return error
}
```

Reply:
makes sense